api/controllers/user.js

```diff
@@ -5,176 +5,175 @@ const jwt = require('jsonwebtoken');
 const User = require('../models/user');
 
 exports.isAdmin = (req, res, next) => {
-    if (req.userData.userType == 'admin')
-        return res.json(true)
-    else
-        return res.json(false)
-}
+  if (req.userData.userType == 'admin') return res.json(true);
```

Comment:
develop a habit of using strict equality ( === ) in comparisons.

Reply:
ok